jupyter_scheduler/orm.py

```diff
@@ -104,6 +105,7 @@ class Job(CommonColumns, Base):
 
 class JobDefinition(CommonColumns, Base):
     __tablename__ = "job_definitions"
+    __table_args__ = {"extend_existing": True}
```

Comment:
I had to do some testing to understand why this is necessary if we are defining `update_db_schema()`. Essentially, this argument is needed to extend the table _metadata_ (e.g. column names & types) associated with the table. If this line is omitted in `test_orm.py`, the test fails with the error:

```
sqlalchemy.exc.InvalidRequestError: Table 'jobs' is already defined for this MetaData instance.  Specify 'extend_existing=True' to redefine options and columns on an existing Table object.
```

So both the table migration (defined in `update_db_schema()`) and table _metadata_ migration (defined here by setting `__table_args__ = {"extend_existing": True}`) must be performed when a new column is added. 😵 

No action needed. I just felt it was necessary to call this out to other readers in this review.